packages/commerce-sdk-react/src/hooks/ShopperBaskets/helpers.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {useCustomerId, useShopperBasketsMutation} from '../index'
+import {useCustomerBaskets} from '../ShopperCustomers'
+import {onClient} from '../../utils'
+import {ApiClients, Argument} from '../types'
+type Client = ApiClients['shopperBaskets']
+
+export function useShopperBasketsMutationHelper() {
+    const customerId = useCustomerId()
+    const {data: basketsData} = useCustomerBaskets(
+        {parameters: {customerId}},
+        {
+            enabled: !!customerId && onClient()
```

Comment:
What is the thought process on only enabling this whilst on the client? I'm asking because Vincent M is working on ideas for allowing personalized content on the server and I'm wondering if we would be placing a limitation on ourselves by adding this now if it's not completely required.

Reply:
@bendvc If I remember correctly, if we allow both server side and client side, it will cause hydration error when client is a registered guest while server will always be a guest user. Hence, we only call this hook on client. 

When Vincent M gets to work on personalization content on server, it will help to mitigate that error. I am not sure what can we do here til we start on that work?

Reply:
What I'm trying to ask is, if we remove the `onClient()` from the condition, will the solution we have effectively behave differently, or is this jsut a precaution? 
My worry is, if we put this condition in now, we can't remove it later. The likelyhood of someone running this on the server is very low, but I guess not zero. So we take the risk and not make it be a breaking change later?